turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -2450,7 +2485,18 @@ async fn resolve_import_map_result(
             {
                 None
             } else {
-                Some(resolve_internal(lookup_path, request, options))
+                let result = resolve_internal(lookup_path, request, options);
+                if let (Some(rewritten), Some(original)) = (
```

Comment:
request_pattern should not return an Option. It can always return a valid pattern